model/fv_regional_bc.F90

```diff
@@ -1319,10 +1319,13 @@ subroutine start_regional_cold_start(Atm, ak, bk, levp            &
 !-----------------------------------------------------------------------
 !
       if (Atm%flagstruct%hrrrv3_ic) then
-        data_source_fv3gfs = .TRUE.
+!        data_source_fv3gfs = .TRUE.
```

Comment:
Do we still need this line in the future? If not, please remove the comment lines.